### ISSUE TITLE
Missing trace on the right side PCB (power switch-> RAW jumper).

### ISSUE THREAD

**Issue by articnet** [pashutk/chocofi#23]
As per picture, if this trace is not connected, the right side does not currently switch on (can be solved by attaching the switch on the other side of the pcb instead and connecting the other side jumper, or by connecting a wire as in the picture). Is this an issue or am I missing something? Thanks!

![pcb_missing_trace](https://github.com/user-attachments/assets/9849be16-f55e-427e-91c1-7b6a983f9d75)

**Comment by TannerGabriel** [comment]
Yes, I just had the same issue.  I could fix the board layout, but it looks like the schematic and board are out of sync. It seems like #9 only modified the board and didn't add the power switch to the schematic.

**Comment by TannerGabriel** [comment]
@pashutk @beta-tank  If possible, could you confirm that the schematic wasn't updated and if so do you still have the updated schematic file?
